scripts/zones/Quicksand_Caves/Zone.lua

```diff
@@ -196,3 +205,21 @@ function onEventFinish(player,csid,option)
     --printf("CSID: %u",csid);
     --printf("RESULT: %u",option);
 end;    
+
+-----------------------------------
+-- UpdateAnticanTagSpawnPoint
+----------------------------------
+
+function UpdateAnticanTagSpawnPoint(id, minTime, maxTime)
+    local npc = GetNPCByID(id);
+    local respawnTime = math.random(minTime, maxTime);
+    local newPosition = npcUtil.pickNewPosition(npc:getID(), anticanTagPositions, true);
+
+    if (GetServerVariable("[POP]Antican_Tag") <= os.time(t)) then
+        npc:hideNPC(1); -- hide so the NPC is not "moving" through the zone
+        npc:setPos(newPosition.x, newPosition.y, newPosition.z);
+    end
+    npc:timer(respawnTime * 1000, function(npc)
+        UpdateAnticanTagSpawnPoint(id, minTime, maxTime);
+    end)
+end;
```

Comment:
derp. leave the timer where it is
just move the function to npcUtil and it's gud